ui/src/components/device/DeviceList.vue

```diff
@@ -161,14 +143,15 @@ export default {
           align: 'center'
         },
         {
-          text: 'Name',
-          value: 'name'
+          text: 'Hostname',
+          value: 'hostname',
```

Comment:
Shouldn't the value be 'name' instead of  'hostname'?

Reply:
In the future, the API will send 'name' or 'hostname'?